examples/hello-grudge.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import pyopencl as cl
+from grudge.discretization import DiscretizationCollection
+import grudge.op as op
+from meshmode.mesh.generation import generate_box_mesh
+from meshmode.array_context import PyOpenCLArrayContext
+from arraycontext import freeze, thaw
+from grudge.dof_desc import DTAG_BOUNDARY
+from meshmode.discretization.connection import FACE_RESTR_INTERIOR
+
+
+ctx = cl.create_some_context()
+queue = cl.CommandQueue(ctx)
+actx = PyOpenCLArrayContext(queue)
+
+
+nel = 10
+coords = np.linspace(0, 2*np.pi, nel)
+mesh = generate_box_mesh((coords,),
+                         boundary_tag_to_face={"left": ["-x"],
+                                               "right": ["+x"]})
+dcoll = DiscretizationCollection(actx, mesh, order=1)
+
+
+def initial_condition(x):
+    # 'x' contains ndim arrays.
+    # 'x[0]' gets the first coordinate value of all the nodes
+    return actx.np.sin(x[0])
+
+
+def left_boundary_condition(x, t):
+    return actx.np.sin(x[0] - 2 * np.pi * t)
+
+
+def flux(dcoll, u_tpair):
+    dd = u_tpair.dd
+    velocity = np.array([2 * np.pi])
```

Comment:
Set the velocity field as a global? It seems to be used in several places.

Reply:
I specifically chose to avoid this to keep the hello-world example as "flat" as possible. My main concern with the other examples in `grudge/examples` was the indirection in control was making it difficult to follow along what was happening.

Reply:
I agree that it's easy to read as is, so I'm fine with it.